src/compiler/scala/tools/nsc/typechecker/Namers.scala

```diff
@@ -581,20 +581,23 @@ trait Namers extends MethodSynthesis {
         }
       }
 
-      def noDuplicates(names: List[Name], check: DuplicatesErrorKinds.Value) {
-        def loop(xs: List[Name]): Unit = xs match {
+      def noDuplicatesBy(namer: ImportSelector => Name, check: DuplicatesErrorKinds.Value): Unit = {
```

Comment:
Given that `namer` is a venerable piece of the local dialect, and especially given the file we're in, I'd prefer this be `rename`d.

Reply:
I thought it was easier than `fun` (for function). Perhaps `byName` would do? 

If you have any clear suggestions, you can use Github's suggestion button.